searx/engines/__init__.py

```diff
@@ -146,11 +146,29 @@ def update_engine_attributes(engine, engine_data):
             setattr(engine, arg_name, copy.deepcopy(arg_value))
 
 
-def set_language_attributes(engine):
+def set_language_attributes(engine, engine_data):
     # pylint: disable=protected-access
     # assign supported languages from json file
+
     if engine.name in ENGINES_LANGUAGES:
         engine.supported_languages = ENGINES_LANGUAGES[engine.name]
+    elif engine.engine in ENGINES_LANGUAGES:
+        # The key of the dictionary ENGINES_LANGUAGES is the *engine name*
+        # configured in settings.xml.  When multiple engines are configured in
+        # settings.yml to use the same origin engine (python module) these
+        # additional engines can use the languages from the origin engine.
+        # For this use the configured ``engine: ...`` from settings.yml
+        engine.supported_languages = ENGINES_LANGUAGES[engine.engine]
```

Comment:
Nitpicking, it can be :
```python
engine.supported_languages = ENGINES_LANGUAGES[engine_data["engine"]]
```

so the change on line 140 is not necessary.

Reply:
I think the reason is redundancy `engine.__name__ == engine.engine`.

But as described, nitpicking.